src/vm/i386/jitinterfacex86.cpp

```diff
@@ -1397,10 +1397,16 @@ void EmitFastGetSharedStaticBase(CPUSTUBLINKER *psl, CodeLabel *init, bool bCCto
         // DoInit:
         psl->EmitLabel(DoInit);
 
-#if defined(UNIX_X86_ABI)
-        // sub esp, 8 ; to align the stack
-        psl->X86EmitSubEsp(8);
-#endif
+#ifdef FEATURE_PAL
```

Comment:
Why don't we have everything under UNIX_X86_ABI? What would be the FEATURE_PAL without UNIX_X86_ABI?